### PR TITLE
183727022 - fix method determining is validator a delinquent

### PR DESCRIPTION
**Pull request by kamtarnowski**
#### What's this PR do?
PR fixes the logic concerning validator activation in the service.

#### How should this be manually tested?
On localhost the following should be performed within the console: `ValidatorCheckActiveService.new.update_validator_activity`

On stage the cron should run the `ValidatorCheckActiveWorker`.

In both cases this validator https://www.validators.app/validators/E2Ydx12gumhPgxcFfxmaz796aC5it3vooZvAJYaaaEkz?locale=en&network=testnet should return status active (if is being voting).

#### What are the relevant tickets?
[- URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/183727022)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?

### CODE REVIEW

File: app/services/validator_check_active_service.rb [hunk 1]
```diff
@@ -86,16 +86,7 @@ def update_scorable(validator)
...
-                                              account: validator.account,
-                                              delinquent: false
-                                            )
-                                            .where(
-                                              'created_at > ?',
-                                              DateTime.now - @delinquent_time
-                                            )
-
-    nondelinquent_history.exists?
+    !validator.delinquent?
```
Comment (maciejkocylapc) [inline app/services/validator_check_active_service.rb:89]: I would merge the two options like `nondelinquent_history.exists? || !validator.delinquent?`

File: test/services/validator_check_active_service_test.rb [hunk 2]
```diff
@@ -67,16 +67,14 @@ class ValidatorCheckActiveWorkerTest < ActiveSupport::TestCase
     assert v.reload.is_rpc
   end
 
-  test 'validator delinquent for too long should be inactive' do
```
Comment (maciejkocylapc) [inline test/services/validator_check_active_service_test.rb:70]: I don't think this test should be deleted

**Review by maciejkocylapc — CHANGES_REQUESTED** [review]

File: app/services/validator_check_active_service.rb [hunk 3]
```diff
@@ -110,4 +102,8 @@ def acceptable_stake?(validator)
     with_acceptable_stake.exists?
   end
 
+  def delinquent_history(validator)
```
Comment (maciejkocylapc) [inline app/services/validator_check_active_service.rb:105]: this method is not the same as the previous one and does not work as it should

**Review by maciejkocylapc — CHANGES_REQUESTED** [review]

File: app/services/validator_check_active_service.rb (hunk 3 above)
Comment (kamtarnowski) [reply app/services/validator_check_active_service.rb]: Fixed.

**Review by maciejkocylapc — APPROVED** [review]

**Pull request merged**